meson/post_install.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+
+schemadir = os.path.join(os.environ['MESON_INSTALL_PREFIX'],
```

Comment:
Do not rely on `MESON_INSTALL_PREFIX` – instead pass `schemadir` to the script. `schemadir` should be `join_paths(get_option('prefix'), get_option('datadir'), 'glib-2.0', 'schemas')`.